File: app/src/main/java/com/nicholasdoglio/eyebleach/model/reddit/Resolution_.java
package com.nicholasdoglio.eyebleach.model.reddit;

import com.squareup.moshi.Json;

public class Resolution_ {

    @Json(name = "url")
    private String url;
    @Json(name = "width")
    private Integer width;
    @Json(name = "height")
    private Integer height;


    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public Integer getWidth() {
        return width;
    }

    public void setWidth(Integer width) {
        this.width = width;
    }

    public Integer getHeight() {
        return height;
    }

    public void setHeight(Integer height) {
        this.height = height;
    }

}

File: remote/src/main/java/com/nicholasdoglio/remote/entities/Resolution.kt
/*
 * MIT License
 *
 * Copyright (c) 2019 Nicholas Doglio
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy
 * of this software and associated documentation files (the "Software"), to deal
 * in the Software without restriction, including without limitation the rights
 * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 * copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice shall be included in all
 * copies or substantial portions of the Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 * SOFTWARE.
 */

package com.nicholasdoglio.remote.entities

import com.squareup.moshi.Json

data class Resolution(
    @Json(name = "url") val url: String,
    // https://i.redditmedia.com/ve8JWirKp-85ZKWYUleUF7oH16jpAjtMm8Oagyz_Ojg.jpg?fit=crop&amp;crop=faces%2Centropy&amp;arh=2&amp;w=1080&amp;s=5316b203fa6bef3b37915500d074b9fc
    @Json(name = "width") val width: Int, // 1080
    @Json(name = "height") val height: Int // 1440
)